.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
I'm sorry but this should be kept.

Without this, we have no guarantee to have test with a classic configuration (means no version tweaks or other).